util/config/gather-cray-prgenv-arguments.bash

```diff
@@ -8,6 +8,8 @@
 # $4 = CHPL_COMM_SUBSTRATE
```

Comment:
Is this file still needed?

Reply:
Yes, it's used in the runtime build, for now at least. Future work to remove it.